web/src/lib/ironclad-activity-api.ts

```diff
@@ -0,0 +1,102 @@
+import getConfig from "next/config";
+
+type LastSignedResponse = Record<string, boolean>;
+
+type GroupDataResponse = {
+  versions: Array<string>;
+  group: number;
+};
+
+export class IroncladActivityApi {
+  private _psAccessId: string;
+  private _psGroupKey: string;
+  private _baseUrl: string;
+  private _isLatestSigned: boolean | null;
+  private _groupData: GroupDataResponse | null;
+
+  constructor() {
+    const { publicRuntimeConfig } = getConfig();
+
+    this._psAccessId =
```

Comment:
this is now only called server side, let's make the env vars server-side only

Reply:
@pdtfh I've updated envs names and also created [PR](https://github.com/worldcoin/world-id-sdk-deployments/pull/71) for the deployment repo